### ISSUE TITLE
Didn't get a boss crate when I killed the boss

### ISSUE THREAD

**Issue by customgamer10** [Iskallia/Vault-public#506]
I killed the vault robot but didn't get a boss crate

**Comment by jmilthedude** [comment]
Hi, is there anything significant in the logs when this happened? Can you specify the version you are on and things like that?

**Comment by SamayaNL** [comment]
Yes, this happened a couple of times now for me as well. Or, I entered the vault with three crates, defeated the boss, exited the vault with three crates, one being the boss crate, so I lost one. Other times I just didnt get the boss crate. The last time I made extra sure to walk around the killing area as well.

**Comment by SamayaNL** [comment]
This happened on 1.6 and 1.6.2 btw

**Comment by customgamer10** [comment]
It happend on 1.6.1 and I don't have access to the logs as I play on one of iskalls servers

**Comment by SamayaNL** [comment]
I play in a Single Player world

**Comment by jmilthedude** [comment]
@customgamer10 can you tell me which server? I have access to the logs and can check if you also give me a time that this happened. otherwise, there is not much I can do.

@SamayaNL If you can find the log of the time that this happened that would be tremendously helpful.

**Comment by SamayaNL** [comment]
@jmilthedude Where can I find the logs? I don't exactly remember when it was though, so I hope I can be of help.

**Comment by jmilthedude** [comment]
@SamayaNL  if you don't remember it might be tough to comb through them. If you remember the day at least, you could go into the .minecraft/logs folder and grb the archive from the day it happened.

**Comment by SamayaNL** [comment]
> @SamayaNL if you don't remember it might be tough to comb through them. If you remember the day at least, you could go into the .minecraft/logs folder and grb the archive from the day it happened.

I think it was on the day I posted my first reply here, so I can check those logs out. I see I have 4 logs for that day. Can you tell me what entry I can look for? Like a specific line for exiting a vault?

**Comment by jmilthedude** [comment]
@SamayaNL you can send them to me and I can go through them. if you're on discord jmilthedude#8509 or jmilthedude@gmail.com

**Comment by SamayaNL** [comment]
@jmilthedude It happened again. I remembered to check the logs this time, and this is what it said at the appropriate time.

> [07:29:22] [Render thread/INFO]: [CHAT] SamayaNL defeated Boss!
[07:29:42] [Server thread/INFO]: EntityJoinWorldEvent fired (playerJoined: true, reloadFired: true, customTagsEventFired: true, vanillaTagsEventFired: true)
[07:29:42] [Render thread/INFO]: EntityJoinWorldEvent fired (playerJoined: true, reloadFired: true, customTagsEventFired: true, vanillaTagsEventFired: true)
[07:34:33] [Server thread/WARN]: Can't keep up! Is the server overloaded? Running 3682ms or 73 ticks behind
[07:36:24] [Render thread/INFO]: [iskallia.vault.client.gui.tab.SkillTab:func_231164_f_:112]: PlayerStatisticsTab closed.
[07:36:28] [Render thread/INFO]: [iskallia.vault.client.gui.tab.SkillTab:func_231164_f_:112]: AbilitiesTab closed.

I can send you the log via email if you want to

Thanks!

PS. Can we give ourselves a bosscrate with a command? I'm guessing not but it would help

**Comment by jmilthedude** [comment]
I'm not seeing anything that would indicate why it would happen in the logs you sent, nor this log. I do think there is a way to give yourself a crate if you are OP, I will check on that. Meanwhile, I will try and test this extensively to replicate.

**Comment by SamayaNL** [comment]
@jmilthedude I think I narrowed it down. I'm not getting bosscrates when I defeat I raffle boss. At least not for the last 3 times.

*edit: scratch that, I did get a boss crate., but lost a normal crate I entered the vault with. I always take care not to miss a crate during the run, and it happened thrice now that on a raffle I'm losing a crate, so I do believe player error is out of the question.
